tests/coalaTest.py

```diff
@@ -322,3 +322,17 @@ def test_coala_without_color(self):
                 self.assertRegex(err, '^\[WARNING\]')
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
+
+    def test_coala_ignore_file(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '-c', os.devnull,
+                    '--non-interactive',
+                    '-f', re.escape(filename),
+                    '-i', re.escape(filename),
+                    '-b', 'LineCountTestBear')
+            self.assertNotIn(stdout, 'This file has 1 lines')
```

Comment:
You can't rely on this. The output of `LineCountTestBear` might change in the future. Then it won't get recognized in this test if for some reason the ignore flag doesn't work correctly anymore

Without ignore, `stdout` is currently like:

```
Executing section cli...

..\..\..\..\zimmer~1\appdata\local\temp\tmp0v9kz559
**** LineCountTestBear [Section: cli | Severity: INFO] ****
!    ! This file has 1 lines.
```

Which can also change in the future. So it's generally not a good idea to only explicitly check for some specific text to not appear in `stdout`

With ignore `stdout` is just:

```
Executing section cli...
```

So better explicitly check that nothing more than the latter appears in `stdout`

@Makman2